lesson15-sims_life/person.py

```diff
@@ -0,0 +1,40 @@
+from random import randint
+from cars import Car
+from bank import BankAccount
+
+
+class Person:
+    def __init__(self, name):
+        self.name = name
+        self.bank_account = BankAccount()
+        self.car = None
+        self.house = None
+
+    def __str__(self):
+        return f"Name - {self.name}, money - {self.bank_account}, car - {self.car}, house - {self.house}"
+
+    def work(self):
+        salary = randint(5, 10)
+        self.bank_account.add_money(salary)
+
+    def buy_car(self, car):
+        if self.bank_account.pay(car.car_price):
+            self.car = True
+            return self.car
+        return False
+
+    def sell_car(self, car):
+        if self.car is True:
+            self.bank_account.sell(car.car_price)
+            self.car = None
+
+    def buy_house(self, house):
+        if self.car is True and self.bank_account.pay(house.house_price):
+            self.house = True
```

Comment:
Also here

Reply:
Not true - should be an instance of House

Reply:
what exactly do I need to change?